src/rbac-forms/formPasswordFragment.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { PureComponent, Fragment } from 'react';
+import { Field } from 'react-final-form';
+import { required } from 'redux-form-validators';
+import { Col } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { FinalFormField } from '../forms';
+import { __ } from '../global-functions';
+
+export default class PasswordFragment extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      editEnabled: false,
+    };
+  }
+
+  handleChangeEdit = () => this.setState(prevState => ({ editEnabled: !prevState.editEnabled }));
+
+  render() {
+    const { isEditing, changeValue } = this.props;
+    const { editEnabled } = this.state;
+    if (!isEditing || editEnabled) {
+      return (
+        <Fragment>
+          <Col xs={12}>
+            <Field
+              id="password"
+              name="password"
+              type="password"
+              validate={required({ msg: __('Required') })}
+              component={FinalFormField}
+              label={__('Password')}
+            >
+              {
+                isEditing &&
+                <Col md={2}>
+                  <button
+                    id="password-change-disabler"
+                    tabIndex="-1"
```

Comment:
Why the negative `tabIndex`? You will not be able to access this button when clicking on `tab` button.

Reply:
Honestly can remember, will remove it